.github/workflows/cd.yml

```
@@ -0,0 +1,38 @@
+name: cd
+on:
+  workflow_run:
+    workflows: ["ci"]
+    types:
+    - completed
```

You add the tag event separately since it is difficult to understand if the event is incoming from a tag
```suggestion
  workflow_run:
    workflows: ["ci"]
    types:
    - completed
  push:
    tags:
      - "v*"
```